app/scripts/modules/netflix/fastProperties/modal/fastPropertyUpsert.controller.js

```diff
@@ -108,7 +109,7 @@ module.exports = angular
     vm.property.targetScope = angular.copy(fastProperty.selectedScope) || {};
 
     vm.strategies = fastPropertyStrategy.getStrategies();
-    vm.selectedStrategy = vm.strategies.length === 1 ? {selected: _.first(vm.strategies)} : {};
+    vm.selectedStrategy = vm.strategies.length === 1 ? {selected: _.head(vm.strategies)} : {};
```

Comment:
`vm.strategies[0]` would read a little easier here, since we already know there's exactly one item.